connectors/http-json/element-templates/uipath-custom-template.json

```diff
@@ -0,0 +1,580 @@
+    },
+    {
+      "id": "timeout",
+      "label": "Connect Timeout"
+    },
+    {
+      "id": "output",
+      "label": "Response Mapping"
+    },
+    {
+      "id": "errors",
+      "label": "Error Handling"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "value": "io.camunda:http-json:1",
+      "binding": {
+        "type": "zeebe:taskDefinition:type"
+      }
+    },
+    {
+      "label": "Type",
+      "id": "authenticationType",
+      "group": "authentication",
+      "description": "Choose the authentication type. Select 'No Auth' if no authentication is necessary",
+      "value": "noAuth",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "No Auth",
+          "value": "noAuth"
+        },
+        {
+          "name": "Basic Auth",
+          "value": "basic"
+        },
+        {
+          "name": "Bearer Token Auth",
+          "value": "bearer"
+        },
+        {
+          "name": "OAuth 2.0",
+          "value": "oauth-client-credentials-flow"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      }
+    },
+    {
+      "id": "method",
+      "label": "Method",
+      "group": "endpoint",
+      "type": "Dropdown",
+      "value": "get",
+      "choices": [
+        {
+          "name": "GET",
+          "value": "get"
+        },
+        {
+          "name": "POST",
+          "value": "post"
+        },
+        {
+          "name": "PATCH",
+          "value": "patch"
+        },
+        {
+          "name": "PUT",
+          "value": "put"
+        },
+        {
+          "name": "DELETE",
+          "value": "delete"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      }
+    },
+    {
+      "label": "Cloud Url",
+      "group": "endpoint",
+      "type": "String",
+      "value": "\"cloud.uipath.com\"",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "cloudUrl"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Cloud Org",
+      "group": "endpoint",
+      "type": "String",
+      "value": "\"testuwxzqtgw\"",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "cloudOrg"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Cloud Tenant",
+      "group": "endpoint",
+      "type": "String",
+      "value": "\"DefaultTenant\"",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "cloudTenant"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "URL",
+      "group": "endpoint",
+      "type": "String",
+      "feel": "required",
+      "value": "=\"https://\"+cloudUrl+\"/\"+cloudOrg+\"/\"+cloudTenant+\"/odata/Queues/UiPathODataSvc.AddQueueItem\"",
```

Comment:
Not sure using process variables is going to be ideal here. Those could also be secret values. We could also be bold and say those HAVE to be secret values for our default template.